devtools/server/actors/replay/connection.js

```diff
@@ -472,18 +472,24 @@ class Recording extends EventEmitter {
   }
 
   _onNewSourcemap(params) {
-    this._lockRecording(params.recordingId);
+    if (params.recordingId) {
```

Comment:
This is how I detect if we're recording to cloud or to disk. Is there a cleaner way?